### ISSUE TITLE
Snowflake Trailing Comma

### ISSUE THREAD

**Issue by ryaminal** [tconbeer/sqlfmt#569]
Snowflake now supports trailing commas. See [this](https://docs.snowflake.com/en/release-notes/2024/8_11#select-supports-trailing-commas) for more info.

example(from the link above):
```sql
select emp_id,
       name,
       dept,
from employees;
```

Curious on what is desired in `sqlfmt` to support this? Maybe nothing needs to be done? I'm not certain.

**Comment by ryaminal** [comment]
mostly just curious if anything needs to be changed from sqlfmt's perspective to support this. didn't want the trailing comma to be removed because it's now valid.
and yeah, i saw the same result and that's why i wasn't certain if anything needs to be done.

**Comment by tconbeer** [comment]
I don't think we need to change anything, but I'll leave this open as a feature request for a "magic trailing comma" feature
